docs/content/CounterLabel.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: CounterLabel
+tags:
```

Comment:
Would you be down to format it with all the tags on the same line like: 
```
tags: pill, other tag, etc
```

Reply:
Sure, I’ll submit an update! 👍

It looks like Doctocat is using Gatsby under the hood, and Gatsby’s  [“Add tags to your `markdown` files”](https://www.gatsbyjs.org/docs/adding-tags-and-categories-to-blog-posts/#add-tags-to-your-markdown-files) uses this format:

```yaml
tags: ["animals", "Chicago", "zoos"]
```

Reply:
Updated in https://github.com/primer/components/pull/586/commits/faee8b0f95d87ea3874399cda87b9363447fd4cb.